style.css

```diff
@@ -174,3 +174,81 @@ hr {
 .lang:hover {
   color: #172b4d;
 }
+
+.contact {
+  background-image: url(images/contact\ form\ background.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-radius: 15% 0% 0% 0%;
+  margin-top: 250px;
+}
+
+.contact-text {
+  color: #252527;
+  margin: 30px;
+  text-align: center;
+  font-size: 23px;
+}
+
+.contact-head {
+  color: #ebebff;
+  padding-top: 40px;
+}
+
+.form {
+  margin: 15px;
+  padding: 20px;
+}
```

Comment:
- Hey, let's improve our design to be close to that of the Figma design below in this form section by aligning the button correctly, and using the correct padding. see the images below:
- Use the correct color on the paragraph of the contact section

*Your design*
![image](https://user-images.githubusercontent.com/59397159/235795534-5542a45d-2ae8-4f67-bece-a4f4ed12b09d.png)


*Figma design*
![image](https://user-images.githubusercontent.com/59397159/235795688-d338c1a3-f485-4ad1-a105-bb8e07accffe.png)